lang/de_DE.po

```diff
@@ -3695,13 +3701,13 @@ msgid "Maximum number of hops must be between 1 and %s"
 msgstr ""
 
 msgid "Maximum number of hops"
-msgstr ""
+msgstr "Maximale Anzahl an Hops"
```

Comment:
"Hops" findet man in der einschlägigen Fachliteratur, e.g. http://www.amazon.de/Br%C3%BCcken-Router-Wege-strukturierten-Netzwerk/dp/3892380511